kyo-stm/shared/src/test/scala/kyo/STMTest.scala

```diff
@@ -476,11 +476,12 @@ class STMTest extends Test:
 
     "Concurrency" - {
 
-        val repeats = 100
+        val repeats = 10
+        val sizes   = Seq(1, 10, 100, 1000)
```

Comment:
I was able to more reliably reproduce the issue in the JVM by increasing the max concurrency of the test to 1000. I've reduced `repeats` so the tests don't take too long